css_tasks/task9/scss/_pricing.scss

```diff
@@ -0,0 +1,120 @@
+@import "variables";
+@import "mixins";
+
+.pricing {
+  background: $mainBackgroundColor;
+  &__container {
+    @include flex;
+    justify-content: space-between;
+    margin: 0 auto;
+    padding: 5rem 0;
+  }
+  &__element {
+    @include flexColumn;
+    align-items: center;
```

Comment:
nothing changes if remove three above, elements go one by one by default, there is no reason to use flex with flex-direction column if you don't change it to row in some media

Reply:
ok